examples/webgl_loader_nrrd.html

```diff
@@ -201,45 +180,12 @@
 
 				controls.update();
 
-				//copy position of the camera into inset
-				camera2.position.copy( camera.position );
-				camera2.position.sub( controls.target );
-				camera2.position.setLength( 300 );
-				camera2.lookAt( scene2.position );
-
 				renderer.render( scene, camera );
-				renderer2.render( scene2, camera2 );
 
 				stats.update();
 
 			}
 
-			function setupInset() {
```

Comment:
Removing this since other loader example don't have a separate axes view.

Besides, the inset should be created like in [webgl_lines_fat](https://threejs.org/examples/webgl_lines_fat) without creating a separate renderer.